lib/modules/datasource/docker/index.ts

```diff
@@ -897,6 +897,26 @@ export class DockerDatasource extends Datasource {
         );
         return this.getTags(registryHost, 'library/' + dockerRepository);
       }
+      // Retry handling for JFrog Artifactory when resolving Docker Official Images
+      // These follow the format of {{registryHost}}{{jFrogRepository}}/library/{{dockerRepository}}
+      if (
+        (err.statusCode === 404 || err.message === PAGE_NOT_FOUND_ERROR) &&
+        dockerRepository.split('/').length === 2 &&
+        registryHost.includes('jfrog.io')
```

Comment:
isn't this better detectable via server response headers?

Reply:
Certainly - in fact that would then also support anyone who runs a custom domain name for their jfrog.io instance 😄 

There are 3 headers we would have access to

- `x-artifactory-id`
- `x-artifactory-node-id`
- `x-jfrog-version`

Any preference?

Reply:
Decided to use `x-jfrog-version`.  The two others look to be correlation ids